### ISSUE TITLE
Rework the mutated living solder

### ISSUE THREAD

**Issue by boubou19** [GTNewHorizons/GT-New-Horizons-Modpack#11506]
### Context
With 2.2.0.0, new alloys were added to the game to substitute the soldering alloy used in nearly if not all recipes involving soldering process.

### Indalloy 140

![image](https://user-images.githubusercontent.com/12850933/193410848-4b6dc121-02c3-41b2-87ae-23eaa1a4e4fe.png)

this is used in nearly all the CAL recipes, proposed in circuits IV+ up to UHV, and in all the LuV+ machine parts (motors, pistons, pumps,....)

Small inconsistency: the quantumprocessor is the only circuit in its familly to have an alternative recipe with indalloy. 

The main issue is that this alloy put yet another pressure on scarse indium stocks. Main way to get indium at IV is to use the chem line with sphalerite, galena, and aluminium. This is a pain until you get void miners (zpm tier) so you can put a void miner on mars to get both galena and sphalerite (roquesite on ross is an absolute joke). Then, once you unlock fusion T3, you can do the fusion recipe, and it's no longer an issue. This means it's an absolute pain to enter LuV because you need tons of indalloy.

a couple of solutions to fix this:
- revert the indalloy back to soldering alloy for anything below ZPM tier
- reduce the amount of indium in the recipe and buff the fluid output

### Mutated living solder
![image](https://user-images.githubusercontent.com/12850933/193412361-cd7e50d8-3795-4fa4-93c0-ec6d0c1916a2.png)

This is used in everything UEV+. The base recipe time seems too high, but with infinity coils, it's a x6 speed boost, so it's actually at 133.3s.
- This is yet another time gate made for the sake of it, i propose to reduce it to like 40s with the infinity coils, so 240s in the base recipe

the fluids in the recipe are weird: how can life live when mixed with fluids at 10000°K (or 9727°C) and at -1200°K (which isn't even a real temperature because absolute 0°K is absolute, can't go below it so this doesn't make sense to use it at all in a "realist" chem line)

- i propose to swap bismuth plasma for molten bismuth (544°K or 273°C) and tin plasma by molten tin (505°K or 232°K). At those temperatures, it makes much more sense to suppose life made its way to support those temperatures in the mixing of the metals. And no need anymore to use the cryotheum.

Another issue is with the recipe quantities. On the recipe displayed above, we can quickly compute that the items contribute for only 300mb in the fluid recipe output. Let's make the assumption that the biocells doesn't count, you still have 8 gravistars (made with 2 ingots of neutronium or 288 mb each) and 2 infinity catalyst dust. Also, 64 biocells in the recipe is too much, stererilized bio catalyst medium is already stressed enough as is in the game, no need to overly consume more of this.

- I propose to let the items to contribute for 2592 mb (2304mb for the gravistars and 288 mb for the infinity catalyst dust)
- I propose to divide the bio cells by 2 to make it 32 cells.

Overall, the new recipe would looks like this:
![image](https://user-images.githubusercontent.com/12850933/193417488-7956ce6b-139b-4059-aa1f-d7f90bbe790f.png)

**Comment by BlueWeabo** [comment]
is it exactly 18000 Tin and Bismut or is it divisible by 144?

The recipe is much more reasonable than before as it can now be much faster. the amount you need much later is insane. Currently we need to even increase the amount we produce to keep up with the demand.

**Comment by boubou19** [comment]
well i can change it to a multiple of 144 for convenience, ez fix.

**Comment by BlueWeabo** [comment]
Huh that is lucky, I didn't know. This will also lower down the requirements to make mutated solder as it will not require any plasmas.

about indalloy. I personally didn't see any problems with it, but that might be from our insane infra at IV.

**Comment by chochem** [comment]
Indalloy:

- Reverting indalloy sounds like an absolutely terrible idea. People just adjusted all their setups. I hope this is a joke?
- In my experience the indium demand is not particularly high compared to other drains. Its just 5% of the alloy.
- There is at least one new way to get indium (bees).
- If Indium is still too scarce I would recommend to buff the milling process some more. E.g. by making the vacuum furnace cheaper and available in LuV.
- The Quantumprocessor ASoC recipe is ZPM, I dont see any inconsistency there. (same goes for nano by the way, which is LuV)

Mutated living solder:
I got less strong opinions here. But when using words like "time gate" you confuse me. With the coils you mention that is one bucket every 3 seconds. That is certainly not a timegate. It might still be insufficient for UMV players, I dont know about that, but a timegate is something different.

**Comment by boubou19** [comment]
> - Reverting indalloy sounds like an absolutely terrible idea. People just adjusted all their setups. I hope this is a joke?

It is not: with the AE2FC change, it's a one click encoding pattern. Not a big issue. I just suggest to revert it to soldering alloy, for LuV only. The fact that no one really looked at those recipes before it went to stable is indeed an issue, but not an excuse to swap it back if needed.

> - In my experience the indium demand is not particularly high compared to other drains. Its just 5% of the alloy.
> - There is at least one new way to get indium (bees).
> - If Indium is still too scarce I would recommend to buff the milling process some more. E.g. by making the vacuum furnace 
cheaper and available in LuV.

That is why i didn"t made any PR yet, as i want feedback from people at those tiers, because i don't master the balance at those tiers, especially with the new indium way from bees. As an UEV player, i just saw the indalloy as a new thing to automate, but i want to make sure people who where about to enter LuV doesn't receive yet another wall in the face after the update.

> The Quantumprocessor ASoC recipe is ZPM, I dont see any inconsistency there. (same goes for nano by the way, which is LuV)

Ok then, but why isn't there new recipes for higher tiers with the fluid? looks like an oversight to me.

> I got less strong opinions here. But when using words like "time gate" you confuse me. With the coils you mention that is one bucket every 3 seconds. That is certainly not a timegate. It might still be insufficient for UMV players, I dont know about that, but a timegate is something different.

This is PWM resource generation: it's as if you made one every 3s, but in reality, you have to wait 133.3s before using all of the newly crafted soldering alloy. That matters a lot in parallel processing.

**Comment by chochem** [comment]
> The fact that no one really looked at those recipes before it went to stable is indeed an issue

That's just blatantly false. Many people have played with this for like 2 months on dev versions (including me). This was not a last minute thing in the slightest. As someone who has played with it from like early zpm on, I would say its completely fine as is or with minor adjustments at most.

> Ok then, but why isn't there new recipes for higher tiers with the fluid? looks like an oversight to me.

can you elaborate? I have no idea what you mean. What new recipes where overlooked?

**Comment by boubou19** [comment]
>>Ok then, but why isn't there new recipes for higher tiers with the fluid? looks like an oversight to me.
>
>can you elaborate? I have no idea what you mean. What new recipes where overlooked?
I am just wondering why is there no UV recipe for the quantum processor assembly for example.

**Comment by chochem** [comment]
so no oversights, ok. Just a proposal.

Making indalloy an option(!) for low tier circuits sounds fine but it's also pretty irrelevant I would say. If someone wants to do it, sure. But there is no reason for it really nor would it have any impact on gameplay in all likelihood.

**Comment by boubou19** [comment]
i just don't understand why only the quantumprocessor was made, but not the other ones. Hence the consitency. But no real issue indeed.

**Comment by chochem** [comment]
LUV+ recipes use the new alloy, very simple and consistent. That includes 2 asoc recipes.

**Comment by xSkewer** [comment]
I played from LuV -> T2 fusion with the solder changes and I never found the indium cost to be that much of an issue. Felt like indalloy was a useless change meant to just piss existing LuV+ players off because my experience was basically the same besides batch crafting indalloy a few times. It never felt challenging or demanding to create (and no I don't support making it any harder or more annoying than it is right now).
Can't comment on living solder since I've never made it.

Wouldn't mind recipe buffs to the solders but changing more assline recipes at this point is just gonna piss more people off so I don't think it should happen.

**Comment by Connor-Colenso** [comment]
Yea I do think we should leave recipes using it alone, anything LuV and above should be using the new alloy and reverting this is just pointlessly annoying players who were playing on dev and switched their systems over. Also it's 5 indium every 100 ingots, really not that much.

Mutated living solder isn't that bad, people just see a big number and get scared. You get at a bare minimum 1 ingot every ~50s, which once you apply upgrades to the chem planet becomes even stronger. I really don't think it's a concern.

**Comment by GDCloudstrike** [comment]
As a person who is crafting the new stargate right now, I can say that living solder is not a concern in any form for me. The recipe time gets quite low with higher tier coils and is never an issue or a "time gate" if you just have it run passively. I personally only have a single chemplant for it running 24/7 and have never run into the issue of running out of it. 
As for the suggestion of changing the plasmas to molten metals, it makes the recipe a lot less interesting as it removes an integral part of it completely. Also, those molten metals are so cheap at that point that they might as well not even be in the recipe as they don't add anything to it. 
Furthermore, the biocell cost reduction does not even impact the recipe cost much, as the current living solder recipe gives enough solder for 7 whole bio mainframes, adding only a cost of 9.14 biocells per, which is a very small addition as the bio mainframes take more than two stacks per already.

**Comment by boubou19** [comment]
alright then
